impl/src/main/java/org/jboss/weld/event/ObserverNotifier.java

```diff
@@ -324,7 +330,7 @@ public RuntimeException apply(Type eventType) {
         }
         // We should always initialize and validate all notification options first
         final NotificationMode mode = initMode(options.get(WeldNotificationOptions.MODE));
-
+        final Long timeout = verifyNotificationTimeoutOption(options);
```

Comment:
Well, then I think `initializeX()` is the best name because initialization usually covers things like validation and verification before the initialized thing is put into service. Whereas `verifyX()` methods usually test whether something has a correct value.